all.json

```diff
@@ -14,6 +14,34 @@
 		"1inch-network.net",
 		"1inch-wallet.org",
 		"1inch.tk",
+		"autoappfix.com",
+		"networkrestoreauth.live",
+		"trasputmang.com",
+		"dappsaccess-bots.net",
+		"imtoken-3w.xyz",
+		"imtoken-gd.xyz",
+		"imtoken-h4.xyz",
+		"imtoken-h9.xyz",
+		"imtoken-p9.xyz",
+		"invest-token.art",
+		"metaalliancedapps.com",
+		"metamask-verify.rf.gd",
+		"metamask.io-toleuhfi.ru",
+		"metamaskapplog.com",
+		"metamaskkeyverif.com",
+		"mymetamaskwallet.ddns.net",
+		"swapdex.tokens-aword.com",
```

Comment:
This is the only one that pops out to me - I see the "connect your metamask", but cannot determine if this is an actual platform on a funds stealer.

Reply:
`https://swapdex.tokens-aword.com/metamask/?nkbihfbeogaeaoehlefnkodbefgpgknn/home#restore-vault` is not a real metamask pop up but a fake one
even if you don't have the metamask plugin it imitates the place where it usually pops up and steals seed phrase

Reply:
![image](https://user-images.githubusercontent.com/49607867/167351655-5e7dcc2c-d48c-4750-9d71-3fe816d6ebf4.png)
this screenshot should suffice ...

Reply:
Brilliant, thanks. We need to have `tokens-aword.com` in the list based on the above.

Reply:
Yup yup, added the mail domain, but didn't bother with all the rest.